build.gradle

```
@@ -30,6 +30,6 @@ tasks.named('test') {
 
 checkstyle {
     toolVersion = '10.2'
-    configFile file('C:\\Users\\user\\Downloads\\namemanagement\\namemanagement\\MyGradleProject\\config\\checkstyle\\checkstyle.xml')
+    configFile file('C:/Users/user/Downloads/namemanagement/namemanagement/docker-tuika/config/checkstyle/checkstyle.xml')
```

こちらですが、相対パスで指定してあげないといけないです！
そうしないと、例えば、私のPCにgit cloneした場合、このディレクトリは存在しないので、エラーになってしまいます！
```suggestion
    configFile file('config/checkstyle/checkstyle.xml')
```

---

あと、`config/checkstyle/checkstyle.xml` っていうファイルが削除されてしまっていないですか？

`/docker-tuika/config/checkstyle/checkstyle.xml`というパスで作ってあげないといけないです！